ReactiveCocoa/UIKit/UIControl.swift

```diff
@@ -0,0 +1,81 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UIControl {
+	/// The current associated action of `self`, with its registered event mask
+	/// and its disposable.
+	internal var associatedAction: Atomic<(action: CocoaAction<Base>, controlEvents: UIControlEvents, disposable: Disposable)?> {
+		return associatedValue { _ in Atomic(nil) }
+	}
+
+	/// Set the associated action of `self` to `action`, and register it for the
+	/// control events specified by `controlEvents`.
+	///
+	/// - parameters:
+	///   - action: The action to be associated.
+	///   - controlEvents: The control event mask.
+	internal func setAction(_ action: CocoaAction<Base>?, for controlEvents: UIControlEvents) {
+		associatedAction.modify { associatedAction in
+			if let old = associatedAction {
+				old.disposable.dispose()
+			}
+
+			if let action = action {
+				base.addTarget(action, action: CocoaAction<Base>.selector, for: controlEvents)
+
+				let disposable = CompositeDisposable()
+				disposable += isEnabled <~ action.isEnabled
+				disposable += { [weak base = self.base] in
+					base?.removeTarget(action, action: CocoaAction<Base>.selector, for: controlEvents)
+				}
+
+				associatedAction = (action, controlEvents, ScopedDisposable(disposable))
+			} else {
+				associatedAction = nil
+			}
+		}
+	}
+
+	/// Create a signal which sends a `next` event for each of the specified
+	/// control events.
+	///
+	/// - parameters:
+	///   - controlEvents: The control event mask.
+	///
+	/// - returns:
+	///   A trigger signal.
+	public func trigger(for controlEvents: UIControlEvents) -> Signal<(), NoError> {
+		return Signal { observer in
+			let receiver = CocoaTarget(observer)
+			base.addTarget(receiver,
+			                   action: #selector(CocoaTarget.sendNext),
```

Comment:
This should be `#selector(receiver.sendNext)` to ensure that the receiver actually has that selector.